pkg/resources/access.go

```diff
@@ -0,0 +1,42 @@
+package resources
```

Comment:
I'm a bit hesistant about `pkg/resources`. All of these things are installation specific. Should we consider `pkg/install` maybe? cc @skriss

Reply:
I think `pkg/install` is more descriptive.